lib/zipkin-tracer/trace.rb

```diff
@@ -0,0 +1,67 @@
+require 'finagle-thrift'
+require 'finagle-thrift/trace'
+
+module Trace
+  class Span
+    def to_h
+      {
+        name: @name,
+        traceId: @span_id.trace_id.to_s,
+        id: @span_id.span_id.to_s,
+        parentId: @span_id.parent_id.nil? ? nil : @span_id.parent_id.to_s,
```

Comment:
ok, I'll see what I can do. I'll be AFAIK until the end of the week though, this is going to be for next Monday.